python/hsfs/feature_view.py

```diff
@@ -100,10 +102,16 @@ def init_serving(
                 If set to False, the online feature store storage connector is used
                 which relies on the private IP.
         """
-        self._vector_server = vector_server.VectorServer(
-            self._featurestore_id, training_dataset_version
-        )
-        self._vector_server.init_serving(self, batch, external)
+        if not batch:
```

Comment:
`batch` can be removed? We should always initiate both instances.

Reply:
aha OK so you mean when ever `init_serving` is called we initiate both? makes sense